src/main/java/net/dv8tion/jda/internal/entities/mentions/AbstractMentions.java

```diff
@@ -255,9 +277,18 @@ public boolean isMentioned(@Nonnull IMentionable mentionable, @Nonnull Message.M
                     return true;
                 break;
             case EMOJI:
-                if (mentionable instanceof Emoji && getCustomEmojis().contains(mentionable))
+                if (mentionable instanceof CustomEmoji && getCustomEmojis().contains(mentionable))
```

Comment:
This seems unrelated? Do we actually want to make this change?

Reply:
`#getCustomEmojis` returns a `List<CustomEmoji>`, it makes no sense to check for (possibly) unicode emojis in a list of custom emojis